_posts/2018-01-17-what-is-on-my-mac-2018.md

```diff
@@ -0,0 +1,79 @@
+
+- [**TextExpander**][5830-0005]:
+I purchased TextExpander back in 2012 and have never regretted it. It saves me a lot of time. Over the years, I've added more and more snippets that prevent me from unnecessary typing. [I even wrote about some of my favorite snippets](https://theboldreport.net/2017/02/my-favorite-textexpander-snippets/). And here's the thing, I love TextExpander not so much for the typing, but for the fact that I don't have to remember specific formats.
+
+- [**iA Writer**][5830-0006]:
+iA Writer is where I do all of my writing on the Mac. I like writing without distraction, and this app makes it possible. Obviously, it's not the only app that promises this, but their built-in preview is great and I love how they format my Markdown as I write. I only wish I could change the font I write in.
+
+- [**Keyboard Maestro**][5830-0007]:
+I'll be honest, I don't get as much out of Keyboard Maestro as others, and as I probably should. Still, to me, it's a critical app. I use it most to manipulate text. For example, it formats Markdown links in the correct way and properly capitalizes my headlines for me. 
+
+- [**Things 3**][5830-0008]:
+I'm in the process of writing a full review of Things 3, but in the meantime, I'll say that I've come to love this app. It's the only todo app I've ever actually used after a few days. I can't pinpoint if it's the app or the stage of life I'm in, but whatever it is, Things 3 helps me get things done and remember to do things I used to forget.
+
+- [**1Password**][5830-0009]:
+1Password is by far my favorite app. It's organized so much of my online life, not to mention secured it. 1Password has lived on my Mac since 2013, and now Kelly and I have a 1Password for Families account.
+
+- [**TripMode**][5830-0010]:
+This is a recent addition to the Mac and was very helpful. TripMode allows you to select what apps have access to the internet, especially useful when you're tethering or on a slow network.
+
+- [**Alfred**][5830-0011]:
+Who knows how long I've been using Alfred, but it's one of the first apps I download on a new Mac. To make a long story short, Alfred helps me input events into my calendar, eject drives, search GIFs, search for RubyGems.org, create slugs, launch apps depending on what task I'm working on, and much more. If you're interested in a separate post about my favorite Alfred extensions, let me know.
+
+- [**Fantastical**][5830-0012]:
+The Calendar app on the Mac is a mess (and on iOS, but that's a separate article). I used BusyCal for a while, and it was ok. In 2012, I didn't feel their app was *that* much better. When Fantastical came about, I fell in love. At the time, just a simple menu bar app that listed the day's events. Fast forward almost six years and the Flexibits team continue to make Fantastical even better.
```

Comment:
"Fast forward almost six years and the Flexibits team continue to make Fantastical even better."

I believe it should be "continues".